### PR TITLE
Stephan/feature/character card component

### PR DESCRIPTION
**Pull request by StephMode**
Add CharacterCard Component and card appending into index.js

### CODE REVIEW

**Comment by akopp90** [comment]
freigegeben

File: index.js [hunk 1]
```diff
@@ -8,6 +12,16 @@ const prevButton = document.querySelector('[data-js="button-prev"]');
...
 
+function renderAndAppendCharacterCard() {
+  // const card = CharacterCard(); // entweder const mit value fn call und dann append() const aufnehmen
+  cardContainer.append(CharacterCard()); // oder einfahc die fn selbst als param in den method
+  // return; // brauchen wir nicht, weil
+  // a) keine Daten aus dieser fn an eine andere weitergeben wollen, wir wollen einfahc nur was appenden
+  // b) in append() schon das return implizit ist
+}
+
+renderAndAppendCharacterCard();
```
Comment (Roland-Hufnagel) [inline index.js:23]: This was not part of the task

File: components/CharacterCard/CharacterCard.js [hunk 2]
```diff
@@ -1 +1,26 @@
-export function CharacterCard() {}
+export default function CharacterCard() {
```
Comment (Roland-Hufnagel) [inline components/CharacterCard/CharacterCard.js:1]: Your function does not have any parameters. Please read the tasks carefully.

**Review by Roland-Hufnagel — COMMENTED** [review]
Please have a look at my comments.

**Pull request merged**